content/docs/configuration.md

```diff
@@ -351,6 +351,9 @@ With this configuration, you'll get builds in all stable fedora releases
 Create a SRPM and submit an RPM build
 to [Fedora Koji](https://koji.fedoraproject.org/koji/) build system.
 
+At the moment it is not possible to run production (non-scratch) builds from upstream.
+For more info, please see [the following issue](https://pagure.io/releng/issue/9801).
```

Comment:
I see a small naming issue in our setup...

We use name *production-build* for koji builds in general and koji uses *production-build* for non-scratch ones..;(

What about this to make it really clear what we mean by this?

```suggestion
At the moment it is not possible to run non-scratch production builds from upstream.
For more info, please see [the following issue](https://pagure.io/releng/issue/9801).
```

You can also append a sentence like "Scratch builds work fine."

Reply:
Ok, changed now